pineappl_cli/src/plot.py

```diff
@@ -1,18 +1,39 @@
 #!/usr/bin/env python3
 
-import math
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 import numpy as np
 import pickle
 
+# global variables coming from the CLI
+# CLI_INSERT_CONFIG_BEGIN
+# add some placeholders meanwhile
+title = ""
+xlabel = ""
+ylabel = ""
+xlog = False
+ylog = False
+scales = 1
+enable_int = False
+enable_abs = False
+enable_rel_ewonoff = False
+enable_abs_pdfs = False
+enable_ratio_pdf = False
+enable_double_ratio_pdf = False
+enable_rel_pdfunc = False
+enable_rel_pdfpull = False
```

Comment:
Put these booleans together, maybe in a dictionary.

Reply:
wait a second: you just told me I should rework so much! and in fact this is a prime example of what I want to rework